spec/migrations/migrate_team_planner_permissions_spec.rb

```diff
@@ -26,8 +26,8 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 
-require Rails.root.join("db/migrate/20220414085531_migrate_team_planner_permissions.rb")
 require 'spec_helper'
+require Rails.root.join("db/migrate/20220414085531_migrate_team_planner_permissions.rb")
```

Comment:
When running the migration by itself, it failed because `Rails` was not defined yet. Switched the require order around